content/blog/2015-08-xx-service-discovery-etc.md

```diff
@@ -0,0 +1,318 @@
+---
+title: Custom service discovery with etcd
+created_at: 2015-06-23
+kind: article
+author_name: Fabian Reinartz 
+---
+
+In a [previous post](/blog/2015-06-01-advanced-service-discovery.md) we
```

Comment:
Got all confused by some links in other sections having the `.md` – alright then.